src/main/docs/guide/azureCosmos.adoc

```diff
@@ -0,0 +1,7 @@
+Micronaut Data Azure Cosmos supports some of the things that are possible to do with JPA and JDBC/R2DBC implementations, including:
```

Comment:
I think we should mention things that aren't supported in comparison to MongoDB/JDBC, like cascading, and it would be nice to have described how the joining work and what the limitations are.

Reply:
There is section I called Azure Cosmos Specifics talking about joins and cascading
https://github.com/micronaut-projects/micronaut-data/pull/1827/files#diff-6e34e60420f3e96d344f9a148edbf6255d92ac27f223f7c67aefe743169c3631

Reply:
Added one sentence and link to the specifics explanation

Reply:
Rephrase to 
`Micronaut Data Azure Cosmos supports some of the features of JPA`